simulation/sub8_simulation/sub8_sim_tools/physics/vehicle.py

```diff
@@ -53,11 +71,43 @@ def __init__(self, world, space, position):
         self.last_cmd_time = time()
         self.set_up_ros()
 
+    def keypressCallback(self, data):
+        """Allows you to manipulate the vehicle through keypress"""
+        #Force
+        if(data.data == "j"):
+            self.body.addRelForce(np.array([-500,0,0]))
+        elif(data.data == 'l'):
+            self.body.addRelForce(np.array([500,0,0]))
+        elif(data.data == 'i'):
+            self.body.addRelForce(np.array([0,500,0]))
+        elif(data.data == 'k'):
+            self.body.addRelForce(np.array([0,-500,0]))
+        elif(data.data == 'u'):
+            self.body.addRelForce(np.array([0,0,500]))
+        elif(data.data == 'o'):
+            self.body.addRelForce(np.array([0,0,-500]))
+
+
+        #Torque
+        if(data.data == "f"):
+            self.body.addRelTorque(np.array([-500,0,0]))
+        elif(data.data == 'h'):
+            self.body.addRelTorque(np.array([500,0,0]))
+        elif(data.data == 't'):
+            self.body.addRelTorque(np.array([0,500,0]))
+        elif(data.data == 'g'):
+            self.body.addRelTorque(np.array([0,-500,0]))
+        elif(data.data == 'r'):
+            self.body.addRelTorque(np.array([0,0,500]))
+        elif(data.data == 'v'):
```

Comment:
Will there ever be a time we will need to move in more than one axis, rotate and translate, rotate in multiple axes, etc at once? using elif would not allow for it.